docs/learn/smart-contract-developers/create-nft-collection.md

```diff
@@ -0,0 +1,168 @@
+---
+sidebar_label: '🗃 Create an NFT Collection'
+sidebar_position: 4
+---
+
+# Create an NFT Collection Using LSP8
+
+This tutorial will explore how to create a collection of unique digital assets.
```

Comment:
Don't forget to add links to other part so readers can explore more.

```suggestion
This tutorial will explore how to create a collection of unique [digital assets](../{..}/standards/tokens/LSP8-Identifiable-Digital-Asset).
```

please adjust the link accordingly